osvr/RenderKit/osvr_display_configuration.cpp

```diff
@@ -530,6 +530,11 @@ void OSVRDisplayConfiguration::parse(const std::string& display_description) {
     {
         auto const& distortion = hmd["distortion"];
 
+        /// Find out how many desired triangles, if this is specified.
+        /// If not specified, set to a default.
+        m_distortionDesiredTriangleCount =
+          distortion.get("desired_triangle_count", 200 * 64).asInt();
```

Comment:
This code follows the rest of the entries that are already in that section of the config file, which use the underscore rather than camelCase. Making this one different would be locally inconsistent. Changing all of them will break existing configuration files.